src/main/java/io/debezium/connector/spanner/db/dao/SchemaDao.java

```diff
@@ -59,13 +58,6 @@ public SpannerSchema getSchema(Timestamp timestamp, Collection<String> tables) {
         return builder.build();
     }
 
-    private boolean isTrue(ResultSet resultSet, int index) {
```

Comment:
@ShuranZhang do you need to rebase this PR back on main?

Reply:
Done.